test/app/helpers/twitter_intent.rb

```diff
@@ -0,0 +1,45 @@
+require 'active_support/core_ext/hash/slice'
+require 'active_support/core_ext/hash/conversions'
+require_relative '../../test_helper'
+require_relative '../../../app/helpers/twitter_intent'
+require 'rack'
+
+class TwitterIntentTest < Minitest::Test
+  TWITTER_URL = 'https://twitter.com/intent/tweet?'.freeze
+
+  def test_no_params
+    assert_equal TWITTER_URL, helper.twitter_share
```

Comment:
It might be better to return nill or an empty string rather than an incomplete url.